### PR TITLE
fix(config): prevent invalid css filenames in webpack output

### LINKED ISSUE #8274: Invalid CSS asset filenames generated, breaks site operation
### Versions

- nuxt: 2.14.17
- node: 10.15.3

### Reproduction

https://github.com/IlyaSemenov/nuxt-css-invalid-filename-reproduction

### Steps to reproduce

Using Node 10.15.3 and yarn 1.22.5:

```
yarn
yarn nuxt build
```

(See README in the reproduction repo.)

### What is Expected?

Build completes successfully. All generated filenames are valid.

### What is actually happening?

Build completes successfully, however one of the filenames is invalid: `pages/123456789_123456789_123456789_123456789_/pages/test1/test/pages/test2/pages/test2/index/pages//6c0ffc7c.c67c3ba.css`. Note the double slash near the end (`pages//6c0ffc7c.c67c3ba.css`).

This invalid filename goes to the manifest. When these static assets are deployed onto certain CDNs (such as Amazon S3 compatible storages), they wouldn't load, for example:

[schools//8be984b7.6b1ee97.css](https://hb.bizmrg.com/eg11-static/pages/user/_role/coverage/pages/user/_role/dashboard/pages/user/_role/operators/_operatorId/schools//8be984b7.6b1ee97.css) -> 403 Access Denied
[schools/8be984b7.6b1ee97.css](https://hb.bizmrg.com/eg11-static/pages/user/_role/coverage/pages/user/_role/dashboard/pages/user/_role/operators/_operatorId/schools/8be984b7.6b1ee97.css) -> 200 OK

Moreover, this broken chunk completely breaks `<nuxt-link>`'s pointing to the affected page! (Not included in the minimal reproduction, although it's not hard to add.)

### Additional context

This is a regression introduced with https://github.com/nuxt/nuxt.js/pull/7463 where the css/images/video URLs were changed to include `[name]`.

Also, the [documentation](https://github.com/nuxt/nuxtjs.org/blob/f36f50ca3555162a1a98933cb705f02ecff3214b/content/en/guides/configuration-glossary/configuration-build.md#filenames) misleadingly lists the old defaults:

```
css: ({ isDev }) => isDev ? '[name].css' : '[contenthash].css'
```

which is not true anymore.

### PR DESCRIPTION
**Pull request by danielroe**
## Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)

## Description
It seems it is possible for `[name]` to include invalid filenames - such as `schools//8be984b7.6b1ee97.css`.

context:  #7463 
closes #8274, #8777

## Checklist:
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly. https://github.com/nuxt/nuxtjs.org/pull/1233
- [x] All new and existing tests are passing.

### CODE REVIEW

File: packages/config/src/config/build.js [hunk 1]
```diff
@@ -16,10 +16,10 @@ export default () => ({
     // { isDev, isClient, isServer }
     app: ({ isDev, isModern }) => isDev ? `[name]${isModern ? '.modern' : ''}.js` : `[contenthash:7]${isModern ? '.modern' : ''}.js`,
     chunk: ({ isDev, isModern }) => isDev ? `[name]${isModern ? '.modern' : ''}.js` : `[contenthash:7]${isModern ? '.modern' : ''}.js`,
-    css: ({ isDev }) => isDev ? '[name].css' : '[name].[contenthash:7].css',
-    img: ({ isDev }) => isDev ? '[path][name].[ext]' : 'img/[name].[contenthash:7].[ext]',
```
Comment (pi0) [inline packages/config/src/config/build.js:20]: We had this from before due to SEO friendliness. I think removing only from css to resolve module naming issue should be enough :)

**Review by pi0 — APPROVED** [review]

**Review by clarkdo — APPROVED** [review]

**Comment by alexcroox** [comment]
Note this is now happening with JS assets too

**Comment by pi0** [comment]
@alexcroox Would you please open a bug report issue? (repro or a hint for repro would be so helpful to quickly address?

**Comment by alexcroox** [comment]
ah shame I was hoping to be able to identify larger bundles in production for splitting opportunities etc. I'll switch to just contenthash.

**Pull request merged**